ethdb/database.go

```diff
@@ -371,7 +372,9 @@ func (b *ldbBatch) Delete(key []byte) error {
 }
 
 func (b *ldbBatch) Write() error {
-	return b.db.Write(b.b, nil)
+	return b.db.Write(b.b, &opt.WriteOptions{
+		Sync: false,
```

Comment:
Isn't this the case by default?

Reply:
https://github.com/google/leveldb/blob/master/include/leveldb/options.h#L188-L210

Reply:
@hackmod yes, `syndtr/goleveldb` appears to just use the default value for bool, which is `false`.

Reply:
I noticed the same, but when I actually experimented with different values, my PC's behavior was noticeably different, better, more responsive, less latency.